benchmark/package.json

```diff
@@ -7,8 +7,11 @@
 	"author": "Biome Developers and Contributors",
 	"license": "MIT OR Apache-2.0",
 	"type": "module",
+	"packageManager": "pnpm@9.6.0",
```

Comment:
Should we require a minimum/specific Node.js version? We don't explain it in the README.md, and I believe we should. The runtime is also important

Reply:
Which tool ensure that we use the correct NodeJS version?

Reply:
the [`engines`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines) fields in the `package.json`